Gemfile

```diff
@@ -177,7 +177,7 @@ gem 'themes_on_rails', '>= 0.3.1', github: 'lowjoel/themes_on_rails',
 
 # Forms made easy for Rails
 gem 'simple_form'
-gem 'simple_form-bootstrap', '>= 1.3.0'
+gem 'simple_form-bootstrap', github: 'allenwq/simple_form-bootstrap'
```

Comment:
PR'ed already, waiting for reviewing, we can wait too.

I am also testing if the exceptions are gone with the fix.